core/rio/languages/src/main/java/org/eclipse/rdf4j/rio/languages/BCP47LanguageHandler.java

```diff
@@ -0,0 +1,101 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Eclipse RDF4J contributors, Aduna, and others.
```

Comment:
I will look for other copyright headers to see if I can fix. That header comes from RFC3066LanguageHandler which I copied to create BCP47LanguageHandler